cloud/cloud_scheduler_test.cc

```diff
@@ -34,7 +33,7 @@ class CloudSchedulerTest : public testing::Test {
         }
       }
       if (running) {
-        usleep(delay);
+        usleep(delaySeconds);
```

Comment:
This is microseconds, not seconds: https://man7.org/linux/man-pages/man3/usleep.3.html